src/resolve-masa.ts

```diff
@@ -18,7 +18,7 @@ export class ResolveMasa implements NameResolver {
     this.masa = masa
       ? masa
       : new Masa({
-          wallet: new providers.JsonRpcProvider(providerUrl).getSigner(),
+          signer: new providers.JsonRpcProvider(providerUrl).getSigner(),
```

Comment:
It looks like the wallet always needs to be provided based on the docs: https://github.com/masa-finance/masa-sdk#configuration.

Reply:
per usage here, seems like wallet is set to a `signer` option. https://github.com/masa-finance/masa-sdk/tree/v3.14.4#usage